server/deploy/helm/charts/lomas_server/templates/dashboard/deployment.yaml

```diff
@@ -0,0 +1,79 @@
+{{- if .Values.dashboard.create -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "lomas.dashboard.fullname" . }}
+  labels:
+    {{- include "lomas.dashboard.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "lomas.dashboard.selectorLabels" . | nindent 6 }}
+  replicas: 1
+  template:
+    metadata:
+      {{- with .Values.dashboard.podAnnotations }}
```

Comment:
Why is it sometimes lomas.dashboard and sometimes .Values.dashboard ? how do you know which one ?

Reply:
- lomas.dashboard is for variables defined in the dashboard/_helpers.tpl file.
- .Values.dashboard is for variables defined by the chart user in the values.yaml file